src/modules/vendor/components/programs/cards/templates/SingleChoiceQuestion.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <div v-if="answersInitialized">
+    <ni-input caption="Question" v-model.trim="card.question" required-field @focus="saveTmp('question')"
+      @blur="updateCard('question')" :error="$v.card.question.$error" type="textarea" />
+    <div class="q-my-xl">
+      <div v-for="(answer, i) in card.answers" :key="i" class="answers">
+        <ni-input :caption="`Réponse ${i + 1}`" v-model.trim="card.answers[i].label" :required-field="i < 2"
+          @focus="saveTmp(`answers[${i}].label`)" :error="answersError(i)" @blur="updateAnswer(i)"
+           :error-message="answersErrorMsg(i)" />
+        <q-radio :val="i" v-model="correctAnswerIndex" @input="updateCorrectAnswer(i)"
```

Comment:
jsp si c'est pertinant, mais ici je crois qu'on execute pas en séquenciel l'update de correctAnswerIndex et le formatage du payload (qui utilise cette variable).
Ptet faudrait remplacer le v-model par value et mettre this.correctAnswerIndex = index; au debut de updateCorrectAnswer